src/python/turicreate/test/test_sframe_sequence_iterator.py

```diff
@@ -12,7 +12,7 @@
 import array
 
 
-IS_PRE_6_0_RC = float(tc.__version__) < 6.0
```

Comment:
@TobyRoseman Is there a more pythonic way to do it?

Reply:
I don't understand. A more pythonic way of doing what?